packages/shared/lib/models/Sync.ts

```diff
@@ -290,13 +290,15 @@ export interface SyncConfigWithProvider {
 export interface IntegrationServiceInterface {
     runScript(
         syncName: string,
+        syncId: string,
         activityLogId: number | undefined,
         nango: NangoSync,
         integrationData: NangoIntegrationData,
         environmentId: number,
         writeToDb: boolean,
         isAction: boolean,
         optionalLoadLocation?: string,
-        input?: object
+        input?: object,
+        temporalContext?: unknown
```

Comment:
why `unknown` and not temporal `Context`

Reply:
updated with https://github.com/NangoHQ/nango/pull/1364/commits/c3da33a3c16ade23b07c92c765565c6cf98f442e